docs/NSR.md

```diff
@@ -90,9 +90,9 @@ Certification Authorities and Delegated Third Parties SHALL:
 
 a.	Implement a Security Support System under the control of CA or Delegated Third Party Trusted Roles that monitors, detects, and reports any security-related configuration change to Certificate Systems;
```

Comment:
Do we need to state that the Security Support System for Root CAs that are Offline and non-networked do not have to be implemented via automated mechanisms?